### ISSUE TITLE
CAPTCHA: When content reflowed it requires two dimensional scrolling

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#171]
**Page:** All (CAPTCHA under Report a problem or mistake on this page)
**URL:** https://recalls-rappels.canada.ca/en
**User Agent:** Chrome (v100)
**Language:** English and French

**Issue Description:** When content is reflowed users need to scroll in both direction to view some content under "CAPTCHA".

![image](https://user-images.githubusercontent.com/12256067/163003929-9eaa1769-8e1e-4e6f-98d5-16fd7ed13891.png)

**Steps to reproduce:**

1. Open the above URL in Chrome
2. Set the desktop browser to 1280px (width) and 1024px (height) and zoom in to 400%. (Chrome -> Web Developer extension -> Resize tab, set the required resize dimensions).
3. Access the "Report a problem or mistake on this page" component

**Expected results:** When content is reflowed, it should be presented in way, users are required to scroll only in one direction to view all content. In this case, it would be vertical scrolling.

**Comment by YuxuanXie01** [comment]
Retested the ticket, issue persists. 
![image](https://github.com/user-attachments/assets/d9ca6959-2617-44e9-aebb-a8f4413a0c79)

**Comment by zimw7** [comment]
Issue still exists.
![image](https://github.com/user-attachments/assets/151a5fa7-d5a1-42d8-a614-d221d7492b8c)

**Comment by 40999221** [comment]
Issue not resolved. Two dimensional scroll bar still exist.

**Comment by echouin** [comment]
From OpenPlus (OP, our vendor that designed the CMS):

Good morning Eric, the width of that box is 300px and is actually provided by reCaptcha (Google). We'd have to override that styling in order to deal with this, but I'm not sure 300px is a realistic screen size. The iPhone5 was 320px wide and was release 12 years ago. I'm sure this is why Google chose this specific size for this box. Are you certain you want us to override this design decision?

**Comment by mmiron-HC** [comment]
@echouin I would tag this as a product limitation if it is the Google Api. The ticket still remains open but we would mark it as escalated (in Jira) to show that it is no longer with you.

**Comment by mmiron-HC** [comment]
@zimw7  Please copy this ticket over to [ART-2605](https://jill.hc-sc.gc.ca/jira/browse/ART-2605) Put the status of this one to Escalated and please include a note to say that this is a google product limitation. Add product_limitation to the labels as well
